src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs

```diff
@@ -71,25 +72,30 @@ public override ExportResult Export(in Batch<Activity> batch)
                             continue;
                         }
 
-                        var array = tag.Value as Array;
-
-                        if (array == null)
+                        if (tag.Value is not Array array)
                         {
                             this.WriteLine($"    {tag.Key}: {tag.Value}");
                             continue;
                         }
 
                         this.WriteLine($"    {tag.Key}: [{string.Join(", ", array.Cast<object>())}]");
                     }
+                }
 
-                    if (!string.IsNullOrEmpty(statusCode))
+                if (activity.Status != ActivityStatusCode.Unset)
+                {
+                    this.WriteLine($"Activity StatusCode : {StatusHelper.GetTagValueForActivityStatusCode(activity.Status)}");
+                    if (activity.Status == ActivityStatusCode.Error)
```

Comment:
We don't need this check. This exporter can simply display the Description if not null.